core/trino-main/src/main/resources/webapp-preview/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
this could live in the directory: `core/trino-main/src/javascript/` rather than in the resources as this will basically bundle sources with the target jar and we don't want that. we just want a build distribution being part of the jar.

Reply:
are you sure?  `src/main/resources` is a filtered directory and only the built assets moving into the target.

![image](https://github.com/user-attachments/assets/09fdf427-fbf3-43a9-9b86-bd820eba297c)